Week-1/Assignment-1/CattyBoard.css

```diff
@@ -0,0 +1,121 @@
+* {
+    margin: 0px;
+    padding: 0px;
+}
+#sidebar {
+    display: flex;
+    grid-template-rows: 20% 20% 20% 20%;
+    align-items: center;
+    margin: 0 0;
+    float: left;
+    width: 60px;
+    padding-bottom: 200px;
+    background-color: #f0f0f0;
+}
+#sidebar .leftnav i {
+    width: 15px;
+    justify-content: space-between;
+    padding-bottom: 50px;
```

Comment:
use consistent padding and margin in all the places also use multiple of 8 for that and keep it in even number not in odd